src/bootstrap/src/core/build_steps/dist.rs

```diff
@@ -1009,6 +1009,8 @@ impl Step for PlainSourceTarball {
                 .arg("--sync")
                 .arg(builder.src.join("./compiler/rustc_codegen_cranelift/Cargo.toml"))
                 .arg("--sync")
+                .arg(builder.src.join("./compiler/rustc_codegen_gcc/Cargo.toml"))
+                .arg("--sync")
```

Comment:
This needs to be mirrored in the instructions given by x.py (see src/bootstrap/bootstrap.py)